pyproject.toml

```diff
@@ -35,22 +35,27 @@ keywords = [
     "setup",
 ]
 classifiers = [
-    "Development Status :: 3 - Alpha",
+    "Development Status :: 4 - Beta",
     "Environment :: Console",
     "Intended Audience :: Science/Research",
     "License :: OSI Approved :: Apache Software License",
-    "Natural Language :: English",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3",
     "Programming Language :: Python :: 3 :: Only",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
```

Comment:
Should we state before testing 3.10/3.11 ?

Reply:
The way we have set the CI pipeline conditions does not allow for manual runs on branches that are not `main` of `hackathon/*`, so it cannot be done before that. However, I did test 3.10 some time ago and it was ok. I doubt 3.11 will be different.